<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifier</class>
 <widget class="QDialog" name="modifier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>-10</y>
     <width>1131</width>
     <height>551</height>
    </rect>
   </property>
   <property name="title">
    <string>Modifier Vol</string>
   </property>
   <widget class="QLineEdit" name="valeur1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="validation">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>450</y>
      <width>193</width>
      <height>28</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QLabel" name="numunique">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> Numéro unique :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="valeur6_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>320</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLabel" name="datedarrive_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>270</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouvelle Date d'arrivée  : </string>
    </property>
   </widget>
   <widget class="QLabel" name="datedepart_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>220</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouvelle Date de départ  : </string>
    </property>
   </widget>
   <widget class="QLabel" name="destdepart_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouvelle Destination de départ : </string>
    </property>
   </widget>
   <widget class="QLabel" name="duree_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>320</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouvelle Durée :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="valeur3_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLabel" name="destarrive_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouvelle Destination d'arrivée : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="valeur5_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>270</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="valeur4_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>220</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="valeur2_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>180</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../vols/ressources pour app qt/238px-Plane_icon.svg.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="valeur7_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>370</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
   </widget>
   <widget class="QLabel" name="duree_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>370</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nouvelle Nombre de passagers :</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>validation</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>modifier</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
